src/client/lazy-app/Compress/index.tsx

```diff
@@ -109,8 +110,12 @@ async function decodeImage(
       if (mimeType === 'image/webp2') {
         return await workerBridge.wp2Decode(signal, blob);
       }
-      // If it's not one of those types, fall through and try built-in decoding for a laugh.
     }
+    // If none of those work, let’s see if Web Codecs API is available
+    if (await WebCodecs.isTypeSupported(mimeType)) {
+      return await abortable(signal, WebCodecs.decode(blob, mimeType));
```

Comment:
I am not returning to the hail-mary path below because Web Codecs API will support all the image formats that built-in canvas supports.

Reply:
Just a slight caveat there, we don't support ICO/CUR and SVG presently. SVG requires DOM, so can't be done in a worker context. ICO/CUR function oddly so we've disabled support in the mean time. Either way the isTypeSupported() will give you a valid answer.

Reply:
Are there any other differences we should be aware of vs `createImageBitmap`?

Reply:
@surma shouldn't this code be rolled into `builtinDecode`? Since it's using the built-in decoders